src/app/services/endpoints.service.ts

```diff
@@ -208,6 +208,14 @@ export class EndpointsService {
     return `${this.jobs}challenge_phase_split/${phaseSplitId}/leaderboard/?page_size=1000`;
   }
 
+  /**
+   * Challenge Leaderboard fetch
```

Comment:
Can you please change this comment here to `Returns or Updates challenge phase split`?